templates/partials/question-definition.html

```diff
@@ -2,13 +2,18 @@
 
 {% for definition in question.definitions %}
   {% call onsCollapsible({
+      "id": question.id ~ "-collapsible",
```

Comment:
I think this will lead to id clashes as there can be more than one definition. Also, to be consistent with the attributes below, this should be `"question-definition-" ~ loop.index`.